Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.7.3
+WORKDIR /bin/app
```

Comment:
could you include a `lable` line in the file, something like this:
```
LABEL maintainer="cgphelp@sanger.ac.uk" \
      uk.ac.sanger.cgp="Cancer, Ageing and Somatic Mutation, Wellcome Trust Sanger Institute" \
      version="3.2" \
      description="telomerecat docker"
```

Pay attention to the version number, it should be the same as what you have in you `setup.py`.